example/make_art_python2.py

```diff
@@ -2,11 +2,13 @@
 
 #code credit goes to: https://www.hackerearth.com/practice/notes/beautiful-python-a-simple-ascii-art-generator-from-images/
 #Extra Task: Make this code work with Python3... you will notice you will have a few errors to fix! If you get stuck, see the answer @ make_art.py
+# Task Done. Code is modified to work with python 3.
```

Comment:
It would be ideal if this file remains in tact for furture students to take a look at. 
Feel free to move your answer out to another file, perhaps in a `v3-Solutions` directory.